README.md

```diff
@@ -38,7 +50,8 @@ Select 1:
 Describe the rules of scope in JavaScript.
 
 Your Answer:
-```text
+1. A variable that is declared outside a function definition is a global variable, and its value is accessible and modifiable throughout your program.
+2. A variable that is declared inside a function definition is local. It is created and destroyed every time the function is executed, and it can not be accessed by any code outside the function.
 ```
```

Comment:
Also, any time you forget the `var` keyword when declaring a variable, it becomes global.